cves/CVE-2018-8014.yml

```diff
@@ -43,7 +43,10 @@ description_instructions: |
   that outsiders to Chromium would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description:
+description: |
+  Default settings for the CORS filter within some versions of Apache Tomcat would 
```

Comment:
Could you explain a bit more on what is a CORS filter?

Reply:
CORS stands for Cross-Origin Resource Sharing, the filter determines if incoming HTTP requests are cross-origin. The filter will then add proper policy and headers to the request before sending it to its intended target.